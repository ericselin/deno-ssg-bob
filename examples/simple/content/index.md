---
title: Index
---

# This is the index
