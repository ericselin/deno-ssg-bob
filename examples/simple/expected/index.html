Index

<h1 id="this-is-the-index">This is the index</h1>
